test/evaluation/test_evaluator.py

```diff
@@ -21,6 +21,46 @@
 QUANTILES = [str(q / 10.0) for q in range(1, 10)]
 
 
+@pytest.mark.parametrize(
+    "target, forecast, expected_metric_values",
+    [
+        (
+            np.array([1.0, 2.0, 3.0, 0.0, 0.0]),
+            np.array([2.0, 3.0, 4.0, 1.0, 0.0]),
+            {
+                Evaluator.mape: (1 + 1 / 2 + 1 / 3) / 3,
```

Comment:
Minor: I would actually write `1 / 1` to make the computation that is performed explicit.